ios/brave-ios/Sources/Brave/WebFilters/CustomFilterListStorage.swift

```diff
@@ -96,8 +97,11 @@ import WebKit
 
   /// Get the file URL to the custom filter list rules regardless if it is saved or not
   private func customRulesFileURL() throws -> URL {
-    let folderURL = try getOrCreateCustomRulesFolder()
-    return folderURL.appendingPathComponent("list.txt")
+    let folderURL = try AsyncFileManager.default.url(
+      for: .applicationSupportDirectory,
+      in: .userDomainMask
+    ).appending(path: "custom_rules")
```

Comment:
This is not detached. If it's not a big issue I can handle that later.

Reply:
This does not require to be async because it does not create a folder